src/Mod/Image/ImageTools/_CommandImageScaling.py

```diff
@@ -32,10 +32,10 @@
     import FreeCADGui
     from PySide import QtGui
     from PySide import QtCore
+    from PySide2 import QtWidgets
```

Comment:
You don't need to add the QtWidgets module, because in FreeCAD it is already included in the QtGui namespace when using Qt5. It doesn't seem to be used here in this commit, so why do you include it?

Reply:
Hi.
Thanks for being vigilant. It is a leftover after playing with a multi monitor setup.
It is fixed now.

Thanks.

Cheers,
Mateusz